tree-sitter-python/TreeSitter/Python/AST.hs

```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE DeriveAnyClass, DeriveGeneric, DeriveTraversable, DuplicateRecordFields, TemplateHaskell, TypeApplications #-}
+{-# LANGUAGE DataKinds, DeriveAnyClass, DeriveGeneric, DeriveTraversable, DuplicateRecordFields, TemplateHaskell, TypeApplications #-}
```

Comment:
We have to have `DataKinds` on here now because we generate e.g. `Token "+" 5` as the definition for anonymous leaves.